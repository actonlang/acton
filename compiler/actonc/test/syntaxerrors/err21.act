# Class definition without colon
class MyClass
    pass