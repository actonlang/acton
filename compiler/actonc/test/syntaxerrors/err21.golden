Building file test/syntaxerrors/err21.act using temporary scratch directory
[error Parse error]: unexpected newline
                     expecting '(', ':', or '['

     +--> test/syntaxerrors/err21.act@2:14-2:15
     |
   2 | class MyClass
     :              ^
     :              `- unexpected newline
     :                 expecting '(', ':', or '['
     :                 
-----+
